IPython/lib/tests/test_pretty.py

```diff
@@ -115,8 +115,8 @@ def test_sets():
     """
     Test that set and frozenset use Python 3 formatting.
     """
-    objects = [set(), frozenset(), set([1]), frozenset([1]), set([1, 2]),
```

Comment:
so here the non literal seem to be on purpose: `Test that set and frozenset use Python 3 formatting` 
Otherwise, we would have tested the round-tripping of the string representation through eval.